automatic/vscode/README.md

```diff
@@ -16,6 +16,7 @@ Build and debug modern web and cloud applications. Code is free and available on
 * `/NoQuicklaunchIcon` - Don't add an icon to the QuickLaunch area.
 * `/NoContextMenuFiles` - Don't add an _Open with Code_ entry to the context menu for files.
 * `/NoContextMenuFolders` - Dont't add an _Open with Code_ entry to the context menu for folders.
+* `/NoAutoUpdate` - Disable Visual Studio Code auto-update feature.
```

Comment:
Maybe also add the switch to the second point in the notes which mentions the separate choco packages to make clear that you can also use package parameter?

Reply:
This should then be added on both places I guess.

And there is a crucial difference. `/NoAutoUpdate` would reapply update on each update, while the package wont. So if you change your mind after installing, the package will not change it as it never gets updated, and package parameter will return it back since it gets remembered which is more semantically correct IMO.